recipes/formulate/LICENSE

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
The license file is distributed with the sdist tarball, so you shouldn't include it here.